test/src/kaltura-player.spec.js

```diff
@@ -616,111 +616,131 @@ describe('kaltura player api', function () {
   });
   describe('async plugins loading', () => {
     let player;
-    let timeStart;
     beforeEach(() => {
       PluginManager.register('asyncResolve', AsyncResolvePlugin);
       PluginManager.register('asyncReject', AsyncRejectPlugin);
     });
 
     afterEach(() => {
+      if (player._pluginManager.get('asyncResolve')) player._pluginManager.get('asyncResolve').destroy();
+      if (player._pluginManager.get('asyncReject')) player._pluginManager.get('asyncReject').destroy();
       PluginManager.unRegister('asyncResolve');
       PluginManager.unRegister('asyncReject');
+      player.destroy();
```

Comment:
this should make the destroy for plugins as well

Reply:
So remove the destroy for plugins via plugins manager

Reply:
good point, done